tests/integration/test_graphql_subscription_transport.py

```
@@ -0,0 +1,434 @@
+    """
+    Validate that the websocket consumer streams subscription updates using the graphql-transport-ws protocol.
+    """
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        class Project(GeneralManager):
+            class Interface(DatabaseInterface):
+                name = CharField(max_length=100)
+
+            class Permission(ManagerBasedPermission):
+                __read__ = ["public"]
```

Ruff (0.14.0)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)